src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -43,6 +44,11 @@ class PeerActor[R <: HandshakeResult](
   import PeerActor._
   import context.{dispatcher, system}
 
+  override val supervisorStrategy: OneForOneStrategy =
+    OneForOneStrategy() {
+      case _ => Stop
```

Comment:
hmm actually.. we should probably also stop PeerActor here. Currently if this happens we end up with a Peer that doesn't have a child connection running. We can just use Escalate here (and PeerManager will stop the whole Peer - see its supervisorStrategy)